plugins/git/git.plugin.zsh

```diff
@@ -34,18 +34,14 @@ function work_in_progress() {
 }
 #change current branch old email with new eamil
 gmvemail() {
-  export GMVEMAIL_GIT_EMAIL_OLD=$1
-  export GMVEMAIL_GIT_EMAIL_NEW=$2
-  git filter-branch --commit-filter '
-    if [ "$GIT_AUTHOR_EMAIL" = "$GMVEMAIL_GIT_EMAIL_OLD" ];
+  OLD_EMAIL=$1 NEW_EMAIL=$2 git filter-branch --commit-filter '
```

Comment:
Just modify as you said.
Thanks for giving me tips to handle temp env var using subshell. Lots of shell knowledge have to learn.